cmd/experimental/kjobctl/pkg/parser/slurm_test.go

```diff
@@ -94,6 +94,7 @@ sleep 10`,
 #SBATCH --time=1:00:00
 #SBATCH --mail-type=END
 #SBATCH --mail-user=your@email.com
+#SBATCH --unknown=unknown
```

Comment:
what is the unknown?

Reply:
In this test, the parser should skip unknown flags and not return an error. In the original version, we used `--time` as an unknown flag, but now it's recognized.